src/cpu/jit_uni_eltwise.cpp

```diff
@@ -608,6 +608,117 @@ void jit_uni_eltwise_injector_f32<isa>::swish_compute_vector(
     h->uni_vmulps(vmm_src, vmm_src, vmm_aux0);
 }
 
+template <cpu_isa_t isa>
+void jit_uni_eltwise_injector_f32<isa>::mish_compute_vector(
+        const Vmm &vmm_src) {
+    // Save src data on stack for later usage
+    h->sub(h->rsp, vlen);
```

Comment:
I bet we have some copypaste from other injectors here, Is it possible to directly call them (e.g. exp_compute_vector) like you do with tanh_compute_vector? Or we have a problem with constant table creation in this case?

Reply:
I use two functions: soft_relu and tanh. But their tables are different. So I made copypaste from soft_relu to here.
I wanted to use exp_compute_vector here like tanh_compute_vector, because there is common table for these injectors. But the exponent function is calculated in exp_compute_vector differently than in soft_relu_compute_vector. I think that this injector (exp_computer_vector) won't work here.